build.zig.zon

```diff
@@ -3,7 +3,7 @@
     .version = "0.0.0",
     .dependencies = .{
         .libFLAC = .{
```

Comment:
IMO the repository is named `flac` so the dependency should also just be named `.flac` not `.libFLAC`